pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -132,7 +132,7 @@ func (trs *TaskRunStatus) GetStartedReason() string {
 }
 
 // GetRunningReason returns the reason set to the "Succeeded" condition when
-// the RunsToCompletion starts running. This is used indicate that the resource
+// the objectWithCondition starts running. This is used indicate that the resource
```

Comment:
nit: the object with condition is `TaskRun` in this function

```suggestion
// the TaskRun starts running. This is used indicate that the resource
```